### DISCUSSION TITLE
Exstra points

### DISCUSSION THREAD

**Discussion by Welatsenn** [IBAX-io/go-ibax#1958] [Rewards]
Bu günlük alınan exstra pointler hakkında bir fikri olan var mı ? Bunları topluyoruz ama sanki genel puanlara yansımıyor gibi !!!

**Comment by Welatsenn** [comment]
Bir bilgisi olan paylaşsın lütfen !!

**Reply by Agnostiko** [reply]
Bu topladığımız puanlar 1 şubata kadarki puana yansımayacak, bunun ödüllendirmesi ayrı olacak diye biliyorum.

**Reply by ByTeachers** [reply]
Günlük puanlar 45000 puanı aşan ve 7 şanslı kişiye dağıtılacak olan 100.000 İBXC için geçerli olacak.
